packages/webapp/src/components/Task/AbandonTask/styles.module.scss

```diff
@@ -0,0 +1,13 @@
+.radioLabel > span{
+    border-radius: 32px;
+    padding: 1px 10px;
+    border: 1px solid var(--grey400);
+    background-color: var(--grey200);
+    color: var(--grey600);
+    margin-left: 5px;
```

Comment:
Do you have access to figma? https://www.figma.com/file/F3f6DpGcxAaEsxfl7OZkI2/Litefarm?node-id=30575%3A100361&t=r61M3p1gOPS6hZMy-0
You can check font-weight and font-size there.

Reply:
Also storybook
https://5fa09c09b8435e0022120d64-apnfscolio.chromatic.com/?path=/story/components-typography--override
https://5fa09c09b8435e0022120d64-apnfscolio.chromatic.com/?path=/story/components-filterpill--selected

Reply:
@JimmyRowland  I tried to use Pill component for this but got following error
<img width="787" alt="Screenshot 2023-02-18 at 7 14 55 AM" src="https://user-images.githubusercontent.com/22586851/219872321-4a759276-dfc3-42fb-8668-ca4b3fe3861f.png">

RadioGroup uses P tag to display its label and Pill uses div and we should not have div inside p. This needs a separate discussion on properly using of tags to display text inside components.

Reply:
You can update radio/radioGroup/pill components to support general patterns like these.

If this date pill pattern only appears here, the pill component can be passed as a prop to each option.

Reply:
Yes, we can, but i think this would be out of scope for this ticket. This issue needs a separate ticket to be addressed.

Reply:
Created tickets:
- https://lite-farm.atlassian.net/browse/LF-2996
- https://lite-farm.atlassian.net/browse/LF-2997